deb/common/control

```diff
@@ -82,7 +82,8 @@ Description: Docker CLI: the open-source application container engine
 
 Package: docker-ce-rootless-extras
 Architecture: linux-any
-Depends: docker-ce, ${shlibs:Depends}
+Depends: ${shlibs:Depends}
+Enhances: docker-ce
 Conflicts: rootlesskit
 Replaces: rootlesskit
 Breaks: rootlesskit
```

Comment:
Looking at these (not for this PR) I'm wondering; should this package `Provides: rootlesskit`, or perhaps the reverse, should `rootlesskit` packages indicate that they `Provides: docker-ce-rootless-extras`

Reply:
I'm not sure. Debian has its own `rootlesskit` package but it does not really provide `docker-ce-rootless-extras`.